src/widget/translator.cpp

```diff
@@ -72,6 +73,11 @@ void Translator::translate()
         QCoreApplication::installTranslator(translator);
     }
 
+    // Added to fix #3542
+    QSet<QString> rtlLanguages = {"ar", "he"};
```

Comment:
Actually, the comment should say that only if language is RTL it needs to be translated to `RTL`, otherwise they don't need to "translate" it.

Also, among qTox langs Uighur appears to be RLT too.